2013/lar-syntactical.json

```diff
@@ -17,7 +17,7 @@
             "property": "actionDate",
             "condition": "call",
             "function": "isActionDateInActivityYear",
-            "args": ["actionDate", "transmittalSheet.activityYear"]
+            "args": ["actionDate", "_HMDA_JSON.hmdaJson.hmdaFile.transmittalSheet.activityYear"]
```

Comment:
I don't like the `_HMDA_JSON.hmdaJson` part of this as it is a hard dependency on internals of the rule-engine. Shouldn't `hmdaFile.transmittalSheet.activityYear` be enough of a path?